Lib/http/cookies.py

```diff
@@ -472,6 +472,9 @@ def __set(self, key, real_value, coded_value):
         dict.__setitem__(self, key, M)
 
     def __setitem__(self, key, value):
+        """Check if the key already exists, return without overwriting"""
```

Comment:
The "dictionary style assignment" is the previous docstring. Please either merge them or remove it. In addition, for Morsels objects, I'm not sure that this should be the expected behaviour. Instead, you should change `__set(key, rval, cval)`.